core/src/main/java/io/grpc/LoadBalancer.java

```diff
@@ -70,10 +70,12 @@ public void shutdown() { }
    *
    * <p>Implementations should not modify the given {@code servers}.
    *
-   * @param servers the resolved server addresses. Never empty.
+   * @param servers the resolved server addresses, sublists are consider equivalent from
+   *                the perspective of the {@link NameResolver}, but can be flattened into a single
```

Comment:
Ideally something like "can be flattened into a single list" and "sublists are consider[ed] equivalent" would be part of the method documentation and not the `@param`. `@param` is best when it doesn't grow too long.

That would also make it easier to go into more detail what is meant by "equivalent." If this is analogous to `EquivalentAddressGroup` we could just say `in the same way as {@link EquivalentAddressGroup}.`